wavelength-plugins/badges.js

```diff
@@ -1,134 +1,148 @@
-/**
- * Badges plugin
- *
- * Credits: Niisama
- *
- * @license MIT license
- */
+/************************
+ * Badges plugin			*
+ *								*
+ * Credits: Niisama		*
+ *								*
+ * Cleanup: Insist		*
+ *								*
+ * @license MIT license *
+ ************************/
 
 'use strict';
 
-const TR_CSS = 'style ="background: rgba(69, 76, 80, 0.8);border: 3px solid #FFF ;border-radius: 4px"';
-const TD_CSS = 'style ="background: rgba(69, 76, 80, 0.6);color: #FFF;padding: 5px;border: 1px solid #222;border: 3px solid #FFF;border-radius: 4px"';
+const TR_CSS = 'style ="background: rgba(69, 76, 80, 0.8); border: 3px solid #FFF; border-radius: 4px"';
+const TD_CSS = 'style ="background: rgba(69, 76, 80, 0.6); color: #FFF; padding: 5px; border: 1px solid #222; border: 3px solid #FFF; border-radius: 4px"';
 
 function badgeImg(IMG_URL, name) {
-	return '<img src="' + IMG_URL + '" height="16" width="16" alt="' + name + '" title="' + name + '" >';
+	return `<img src="${IMG_URL}" height="16" width="16" alt="${name}" title="${name}">`;
 }
 
 exports.commands = {
 	badge: 'badges',
-	badges: function (target, room, user) {
-		let parts = target.split(',');
-		let cmd = parts[0].trim().toLowerCase();
-		let targetUser;
-		let selectedBadge;
-		let userBadges;
-		let userid;
-		let output = '';
-		switch (cmd) {
-		case 'give':
-		case 'set':
+	badges: {
+		give: function (target, room, user) {
+			let targetUser;
```

Comment:
You can avoid the hell of all of these `let` declarations by introducing blocks around the body of the cases, which means that each case will be in its own scope.

For example,
```js
let foo = 'bar';
switch (foo) {
case 'foo': {
   let bar = 'bar';
   console.log(baz);
}
case 'bar': {
   let bar = 'baz';
   console.log(`foo bar ${bar}`);
}
}
```